### ISSUE TITLE
Multiline Kirbytext not working

### ISSUE THREAD

**Issue by movingwater** [getkirby/editor#179]
If I have multiple lines for a KirbyText block it doesn't work as expected.  I can add 
```(link: test)```
which works. Also 
```(link: test) (link: test2)```
But if I put a linebreak in between, it's not working anymore.
```
(link: test)
(link: test2)
```

outputs: 

```
[{"attrs":[],"content":"domain.com/test
domain.com/test2","id":"_y4z0a2epf","type":"kirbytext"}]
```

For this example I could use two separated blocks but If I wan't to use the columns plugin in a block, that's not possible.

```
(columns...)

Column 1

++++

Column 2

(...columns)
```

It looks like this:
```
[{"attrs":[],"content":"

Column 1

Column 2

","id":"_y4z0a2epf","type":"kirbytext"}]
```

**Comment by neildaniels** [comment]
@movingwater Are you using this within a Structure field?

I'm running into this issue as well, but only when the `editor` field is used within a `structure` field.

**Comment by neildaniels** [comment]
This is also an issue for Code blocks, not just Kirbytext.

**Comment by movingwater** [comment]
Yes its, but not structure field but builder field (i use the builder plugin). Sorry for not mentioning this.
